README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My name is *Kathleen Cannell*. I just finished my second year in the college.
+I am majoring in Economics:
+![Economics](https://i.kym-cdn.com/entries/icons/facebook/000/029/959/Screen_Shot_2019-06-05_at_1.26.32_PM.jpg) 
+
+I hope to minor in Inequality, Social Problems, and Change.
+
+**Background**
+I am from [Columbus, Ohio](https://en.wikipedia.org/wiki/Columbus,_Ohio).
+
+**Current Information**
+I am back home in Columbus, and am in the Summer Institute for Social Research Methods.
+I am excited to start research with Professor Waldo Johnson Jr. of SSA.
+
+**Interests**
+I love music and concerts of all kinds. Last year I got to see Clairo, Logic, Khalid, and [Dominic Fike](https://www.youtube.com/watch?v=sMYcubNOrPs), along with a few smaller shows.
+I also enjoy reading, knitting, and being outside.
```

Comment:
Maybe you can try to use a list here( it might not be necessary, but it will be a good try :) )